docs/installation/gcp.md

```diff
@@ -0,0 +1,197 @@
+---
+sidebar_position: 4
+---
+
+# GCP Install Guide
+
+
+## Automatic Install
```

Comment:
I would avoid splitting into an "automatic" and "advanced" section. I don't see a reason to use docker now that we are moving away from terraform. The primary tool that the user will need is `gcloud` and they will need that installed regardless of whether they use docker or not.

I think each install page could just:

1. list any required quotas / etc for the cloud account
2. list bulletpoints of all things that are installed (cluster, bucket, registry)
3. embed the install script directly into the markdown to show the steps